cli/src/main/java/org/apache/falcon/cli/FalconExtensionCLI.java

```diff
@@ -158,6 +162,7 @@ public Options createExtensionOptions() {
         Option resume = new Option(FalconCLIConstants.RESUME_OPT, false, "Resume an extension job");
         Option delete = new Option(FalconCLIConstants.DELETE_OPT, false, "Delete an extension job");
         Option unregister = new Option(FalconCLIConstants.UREGISTER, false, "Delete metadata of extension job");
```

Comment:
This can also have a better description. "Un-register an extension. This will make the extension unavailable for instantiation".